lib/ffprober/ffmpeg/version_validator.rb

```diff
@@ -0,0 +1,25 @@
+module Ffprober
+  module Ffmpeg
+    class VersionValidator
+      MIN_VERSION = Gem::Version.new('0.9.0')
+      MAX_VERSION = Gem::Version.new('2.6.2')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.